src/calculateRentalCost.js

```diff
@@ -29,6 +29,22 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const daysForSmallDiscont = 3;
```

Comment:
Typo in the variable name: 'daysForSmallDiscont' should be 'daysForSmallDiscount'